setup.py

```diff
@@ -106,20 +101,19 @@ def run(self):
     url="https://github.com/mongodb-labs/winkerberos",
     keywords=["Kerberos", "SSPI", "GSSAPI"],
     install_requires=[],
-    test_suite=test_suite,
+    test_suite="test",
     tests_require=tests_require,
     platforms="Windows",
     license="Apache License, Version 2.0",
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
```

Comment:
This is great, but only works with relatively recent versions of setuptools and pip. Can you also add a check for Python version closer to the beginning of this file for version >= 2.7 and raise RuntimeError if not? Somethinfg like this:

https://github.com/mongodb/mongo-python-driver/blob/master/setup.py#L8-L9

Reply:
Good idea, added!